mobile/lib/models/kya.dart

```diff
@@ -83,24 +82,40 @@ class Kya extends Equatable {
   }
 
   @override
-  List<Object> get props => [id];
+  List<Object> get props => [id, title, progress];
```

Comment:
Hi @BenjaminSsempala I think id is sufficient because its the unique identifier.

Reply:
Done